upf/upf.c

```diff
@@ -73,6 +73,8 @@ vnet_upf_ue_ip_pool_add_del (u8 * identity, u8 * nwi_name, int is_add)
   upf_ue_ip_pool_info_t *ueip_pool = NULL;
   uword *p;
 
+  identity = vec_dup (identity);
```

Comment:
Replace another `vec_dup (identity)` with `identity` in the code below, otherwise we're doing `vec_dup()` twice and leaking memory